testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7538,8 +7610,8 @@ private static void checkIf(SqlOperatorFixture f) {
   @Test void testSafeNegateFunc() {
     final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.SAFE_NEGATE);
     f0.checkFails("^safe_negate(2)^",
-                  "No match found for function signature "
-                    + "SAFE_NEGATE\\(<NUMERIC>\\)", false);
+        "No match found for function signature "
```

Comment:
unfortunately indents changed here again lol

Reply:
I had it incorrectly indented from a previous PR so I corrected here to indentation level of 4

Reply:
nvm discussed offline, these are correct